openshift/ci-operator/Dockerfile_with_kodata.in

```diff
@@ -4,7 +4,10 @@ FROM registry.ci.openshift.org/openshift/release:golang-1.18 as builder
 COPY . .
 RUN make install test-install
 
-FROM openshift/origin-base
+FROM registry.access.redhat.com/ubi8/ubi-minimal
+
+RUN microdnf install tzdata
```

Comment:
@pierDipi  added this to the midstream images, to match production (remember we had this PingSource TZ bug).

Question regarding the "template", this is not yet something we generate w/ the `hack` repo?

Reply:
We're use hack for 1.8+